base/applications/mscutils/servman/control.c

```diff
@@ -27,6 +28,7 @@ DoControlService(LPWSTR ServiceName,
     DWORD MaxWait;
     DWORD ReqState, i;
     BOOL Result;
+    DWORD dwResult = SC_MANAGER_SUCCESS;
```

Comment:
The "SC_MANAGER_SUCCESS" error code name is unneeded, you can use the standard Win32 one called "ERROR_SUCCESS" which has the very same value (== 0).

Reply:
Good point. I fix it.